include/bout/mesh.hxx

```diff
@@ -619,10 +642,10 @@ class Mesh {
   /// Calculates the size of a message for a given x and y range
   int msg_len(const vector<FieldData*> &var_list, int xge, int xlt, int yge, int ylt);
   
-  // Initialise derivatives
+  /// Initialise derivatives
   void derivs_init(Options* options);
   
-  // Loop over mesh, applying a stencil in the X direction
+  /// Loop over mesh, applying a stepppncil in the X direction
```

Comment:
typo?

Reply:
Probably me misfingering `C-p` in Emacs...